SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -372,6 +389,73 @@
   <!-- End of FxCop section-->
   <!-- **************************************************************************** -->
 
+  <!-- **************************************************************************** -->
+  <!-- Roslyn analysis section -->
+  <!-- **************************************************************************** -->
+  <!-- Set the properties required to run Roslyn analysis as part of the build. 
+       Note: Roslyn analysis is run using the CSC task in MSBuild 14.0 onwards.
+       Analysis is controlled by the following properties:
+         ResolvedCodeAnalysisRuleSet - the full path to the ruleset to use
+         ErrorLog - the file path for the error reporting file
+         
+       We want to run after the core "ResolveCodeAnalysisRuleSet" target which
+       assigns a value to "ResolvedCodeAnalysisRuleSet".
+  -->
+  <Target Name="OverrideRoslynCodeAnalysisProperties"
+        Condition=" $(SonarQubeTempPath) != '' "
+        DependsOnTargets="SonarQubeCategoriseProject"
+        AfterTargets="ResolveCodeAnalysisRuleSet"
+        BeforeTargets="CoreCompile">
+
+    <PropertyGroup>
+      <!-- Check if the ruleset exists -->
+      <SonarQubeRoslynRulesetFullName>$(SonarQubeConfigPath)SonarQubeRoslyn-cs.ruleset</SonarQubeRoslynRulesetFullName>
+      <SonarQubeRoslynRulesetExists>$([System.IO.File]::Exists($(SonarQubeRoslynRulesetFullName)))</SonarQubeRoslynRulesetExists>
+
+      <!-- Run FxCop only if the ruleset exists (else it means no Roslyn rules are enabled) -->
```

Comment:
@dbolkensteyn Yep. I've corrected it here and in lines 421 and 424 as well (and fixed a "Rolsyn" typo). It'll be part of my next PR.